x-pack/filebeat/module/activemq/module.yml

```diff
@@ -0,0 +1,5 @@
+dashboards:
+- id: Filebeat-activemq-log-Dashboard
+  file: Filebeat-activemq-log.json
+- id: Filebeat-activemq-audit-Dashboard
+  file: Filebeat-activemq-audit.json
```

Comment:
Not sure if this content is used... but these dashboards don't seem to be included.

Reply:
You're right, they were autogenerated. Will come in the next PR, I can remove them for now.

Reply:
Removed.